_plugins/auto-anchor.rb

```diff
@@ -0,0 +1,29 @@
+## Automatically add id tags to list items that are formatted like one
+## of the following:
+# - **<Summary:** Details
+# - **[Summary][]: Details
+# - **[Summary](URL): Details
+Jekyll::Hooks.register :documents, :pre_render do |post|
+  ## Don't process documents if YAML headers say: "auto_id: false"
+  if post.data["auto_id"] != false
```

Comment:
Since this file is generally using single-quotes, which can make sense when you don't need string interpolation or special symbols, single quotes can be used here.